### ISSUE TITLE
AlaSQL doesn't export the globality of data in xls.

### ISSUE THREAD

**Issue by fsebbah** [agershun/alasql#694]
Hi, 
I create a sample here: http://jsfiddle.net/fsebbah/tafqcg2k/
It's an json array. The first record has 4 elements and the second has 8 elements.
The issue, I retrieve only 4 elements in the xls file.
Thanks,
Franck

**Comment by mathiasrw** [comment]
Hvad you send the wrong link? I see only:

```js
  var data = [{city:"Minsk",population:100000}, {city:"Riga",population:200000}];
```

Can you explain the word "Globality" in the title?

**Comment by fsebbah** [comment]
Exact, the link was wrong : http://jsfiddle.net/fsebbah/tafqcg2k/3/

**Comment by mathiasrw** [comment]
the field "data" was not declaired. 

I have updated you example: http://jsfiddle.net/tafqcg2k/4/ and im getting header + 10 rows of data. 

CLosing the issue. Please reopen if any more questions.

**Comment by fsebbah** [comment]
Hi,
Sorry to lost your time. The problem is following:

At the first line, there are **4 elements**:
{"startDate":"2016-06-01","startTime":"07:08","name":" JEROME","matricule":"1010223"}

And the second line, there are **8 elements**
{"startDate":"2016-06-01","startTime":"07:08","endDate":"2016-06-01","endTime":"11:51","indexationHours":8,"indexationTimeMorning":" 04:20","indexationExtraHoursMA":" 00:20","indexationTimeAfternoon":"-","indexationExtraHoursAM":"-","name":" JEROME","matricule":"1010223"},

If you run the sample, Alasql,  **we retrieve only the 4 fields** for the totality of lines and not 
4 at the first line,
8 at the second line, .....

Thanks,
Franck

**Comment by mathiasrw** [comment]
Ahh, ok - I get it now. 

It should work with the `alasql.options.columnlookup` but it does not: 

http://jsfiddle.net/tafqcg2k/5/

**Comment by MatteoFe** [comment]
same problem here, any news?
